.gitignore

```diff
@@ -23,3 +23,4 @@ decko*gem
 *.sqlite3
 *.DS_Store
 *.swp
+test_work/*
```

Comment:
The idea is to ignore working script changes and explicitly add them.  I guess you may have to disable this line to do it, but I expect it won't be that common of an action.